server/postgreSchema.sql

```diff
@@ -0,0 +1,23 @@
+CREATE SCHEMA ReservationsModule;
+
+CREATE TABLE ReservationsModule.Restaurants (
+  id int not null unique,
+  name str,
+  open time,
+  close time,
+  num_seats int
```

Comment:
Int has a range of -2,147,483,648 to 2,147,483,647, if your value is between  -32,768 to 32,767, you should use SMALLINT, which will save time and space!